go-runtime/compile/schema.go

```diff
@@ -257,6 +257,21 @@ func parseConfigDecl(pctx *parseContext, node *ast.CallExpr, fn *types.Func) {
 			Type: st,
 		}
 	}
+
+	// Check for duplicates
+	for _, d := range pctx.module.Decls {
+		c, ok := d.(*schema.Config)
+		if ok && c.Name == name && c.Type.String() == st.String() {
+			pctx.errors.add(errorf(node, "duplicate config declaration"))
```

Comment:
Let's report the location of the other declaration in the message (here and below).

Reply:
great idea - done